Automattic/wp-calypso
# Earn: Fix Extensive Icon Size on Mobile Devices

#### Changes proposed in this Pull Request

* Fixes the size of the "Collect payments for content" icon on mobile

#### Testing instructions

You can verify the change by resizing your screen on the Earn page.

| Before  | After |
| ------------- | ------------- 
<img width="418" alt="Screenshot 2020-07-05 at 09 35 05" src="https://user-images.githubusercontent.com/43215253/86528828-b4c55780-bea3-11ea-84ea-ede5a28992eb.png"> | <img width="414" alt="Screenshot 2020-07-05 at 09 34 50" src="https://user-images.githubusercontent.com/43215253/86528831-b858de80-bea3-11ea-858d-d318a74f5a22.png">
 
cc @mmtr, @artpi 

Fixes #43897
